Foundation/NSPropertyList.swift

```diff
@@ -113,6 +113,8 @@ internal func _expensivePropertyListConversion(input : AnyObject) -> Any {
         return data
     } else if let number = input as? NSNumber {
         return number
+    } else if let url = input as? NSURL {
```

Comment:
@radex do you mind taking on that change as well? With that we should remove the requirement to convert NSURL in the NSPropertyList method.